capstone-frontend/src/components/Login.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react'
+import { GoogleLogin, GoogleLogout } from 'react-google-login';
+import { Dropdown, DropdownButton, ButtonGroup } from 'react-bootstrap'
+import "../styles/Login.css"
+
+
+export class Login extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isLogin: false,
+      profileObj: {},
+      profileMenuCollapsed: false
+    };
+  }
+
+  toggleProfileMenu = () => {
+    this.setState({ profileMenuCollapsed: !this.state.profileMenuCollapsed });
+  }
+
+  loginResponseSuccess = (response) => {
+    this.setState({ isLogin: true });
+    this.setState({ profileObj: response.profileObj });
+
+    console.log("ProfileObj", this.state.profileObj);
+  }
+
+  logoutResponseSuccess = () => {
+    this.setState({ isLogin: false });
+    this.setState({ profileObj: {} });
+  }
+
+  render() {
+    return (
+      <div id="login">
+        {this.state.isLogin
+          ?
+          <div>
+
+            <DropdownButton as={ButtonGroup} title={
```

Comment:
What is the "as={ButtonGroup}" doing?

Reply:
I think is setting the styling and DOM structure of the ```DropdownButton``` based on ```ButtonGroup```.

[ButtonGroup Documentation](https://react-bootstrap.github.io/components/button-group/)

[DropdownButton Documentation](https://react-bootstrap.github.io/components/dropdowns/#dropdown-button-props)

![image](https://user-images.githubusercontent.com/53728084/86066756-50803f00-ba30-11ea-986e-59940e3304be.png)